samples/saga/sagafinder-process.include.md

```diff
@@ -1 +1 @@
-It is important to note that the saga is not sending the order ID to the payment processor. Instead, it is sending a payment transaction ID. A saga can be correlated with messages using more than one property. For example, `OrderId` and `PaymentTransactionId`. This requires both of these properties to be treated as unique.
\ No newline at end of file
+It is important to note that the saga is not sending the order ID to the payment processor. Instead, it is sending a payment transaction ID. This saga needs to be correlated by more than one attribute. For example, `OrderId` and `PaymentTransactionId`. This requires both of these properties to be treated as unique.
```

Comment:
There is a mix of "attribute" and "property" here. What term do we use for these things in the rest of the saga documentation? We should be consistent.

Reply:
property is the correct one. Fixed.